test/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "types": [
+      "expect.js",
```

Comment:
@justinkambic this and the line below require `@types/expect.js` and `@types/mocha` to be added to the root package.json file for the repo.

Reply:
@spalger just wanted to check before I did anything else, I added those to `package.json` in https://github.com/elastic/kibana/pull/29667/commits/472e422c53443a82f9444b4957ae1708c8fd2663, but that caused many type conflicts, see link below. Should I fix those in this PR, or is there some additional configuration we need to do?
https://kibana-ci.elastic.co/job/elastic+kibana+pull-request/767/JOB=x-pack-intake,node=immutable/console

Reply:
I just reopened https://github.com/elastic/kibana/pull/30190 and will work to merge that so we have support for mocha types and I'll extend that to support expect.js types too.

Reply:
Thanks, sorry for opening a can of worms.

Reply:
No worries, if you want to take a look at #30190 that would help move it along